### DISCUSSION TITLE
error when installing mariadb

### DISCUSSION THREAD

**Discussion by Stockhauz** [tteck/Proxmox#1289] [Help Needed] [answered]
Hi, first off i just want to say thanks for all these amazing scripts/guides!

Im having trouble installing mariadb though. keep getting this error when running the install script.
tried 3 times now on different occasions incase there was some problem on the server providing the download, but i still keep getting the same error.
see console output below, any thoughts?

`Using Default Settings
Using Container Type: 1
Using Root Password: Automatic Login
Using Container ID: 105
Using Hostname: mariadb
Using Disk Size: 4GB
Allocated Cores 1
Allocated Ram 1024
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway Address: Default
Disable IPv6: No
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server Address: Host
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a MariaDB LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using VMs1 for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 105 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: <private>
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.121.3
 ✓ Updated Container OS
 ✓ Installed Dependencies
 - Installing MariaDB...curl: (22) The requested URL returned error: 404 

[ERROR] in line 22: exit code 127: while executing command "$@" > /dev/null 2>&1`

**Comment by tteck** [comment]
No issues here

```
    __  ___           _       ____  ____ 
   /  |/  /___ ______(_)___ _/ __ \/ __ )
  / /|_/ / __  / ___/ / __  / / / / __  |
 / /  / / /_/ / /  / / /_/ / /_/ / /_/ / 
/_/  /_/\__,_/_/  /_/\__,_/_____/_____/  
 
Using Advanced Settings
Using Container Type: 1
Using Root Password: Automatic Login
Container ID: 112
Using Hostname: mariadb
Using Disk Size: 4
Allocated Cores: 1
Allocated RAM: 1024
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a MariaDB LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 112 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.253 
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.113.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Installed MariaDB
Would you like to add Adminer? <y/N> y
 ✓ Installed Adminer
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!
```

https://github.com/tteck/Proxmox/issues/914#issuecomment-1381763061

**Comment by Stockhauz** [comment, marked as answer]
Yeah its working fine now for me too now, must have been something temporary with the download of mariadb.
Thanks for the quick reply and sorry to have bothered you :)
